pep-0698.rst

```diff
@@ -320,10 +320,10 @@ Limitations of setting ``__override__``
 As described above, adding ``__override__`` may fail at runtime, in which
 case we will simply return the argument as-is.
 
-In addition, even in cases where it does work it may be difficult for users
-to correctly work with multiple decorators, because getting the ``__override__``
-field to exist on the final output requires understanding the implementation
-of each decorator:
+In addition, even in cases where it does work it may be difficult for users to
```

Comment:
```suggestion
In addition, even in cases where it does work, it may be difficult for users to
```

Nitpick: Needs a comma here to separate these two clauses

Reply:
Mm, I saw this but left it thinking it was a stylistic choice.

Reply:
Same for my first pass as I've become a lot more conservative about suggesting others add commas in PEPs lately, but it is more than merely stylistic in this case.